File: JavaScript  - Introduction to Programming/1. First Steps in Programming/1. First Steps in Programming/2.1. Simple Calculations/12. Currency Converter.js
function currencyConverter([money, currencyOne, currencyTwo]) {
  const valutObj = {'BGN' : 1, 'USD' : 1.79549, 'EUR' : 1.95583, 'GBP' : 2.53405}
  return  (money * valutObj[currencyOne] / valutObj[currencyTwo]).toFixed(2) + ' ' + currencyTwo
}

// console.log(currencyConverter([20, 'USD', 'BGN']))
// console.log(currencyConverter([150.35, 'USD', 'EUR']))

File: Pyrhon - Introduction to Programming/2.1. Simple Calculations/12. Currency Converter.py
money, currency_one, currency_two = [input() for _ in range(3)]
valut_dict = {'BGN' : 1, 'USD' : 1.79549, 'EUR' : 1.95583, 'GBP' : 2.53405}

print(f'{round(float(money) * valut_dict[currency_one] / valut_dict[currency_two], 2)} {currency_two}')



